meta-mentor-staging/openembedded-layer/recipes-extended/libmodbus/libmodbus_%.bbappend

```diff
@@ -0,0 +1,17 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/libmodbus:"
+SRC_URI += "\
+    file://run-ptest \
+    file://local-tcp-test \
+"
+
+inherit ptest
+
+# do_compile_ptest() {
+#         oe_runmake SUBDIRS=tests unit-test-server unit-test-client
+# }
```

Comment:
Oops, forgot this was still there :)

Reply:
And I did not even notice the hashes :smirk: